src/EventStore.Core/Services/Transport/Grpc/Gossip.Read.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using EventStore.Client.Gossip;
+using EventStore.Client.Shared;
+using EventStore.Core.Messages;
+using EventStore.Core.Messaging;
+using EventStore.Plugins.Authorization;
+using Grpc.Core;
+using static EventStore.Common.Utils.EndpointExtensions;
+using ClusterInfo = EventStore.Client.Gossip.ClusterInfo;
+using MemberInfo = EventStore.Client.Gossip.MemberInfo;
+
+namespace EventStore.Core.Services.Transport.Grpc {
+	partial class Gossip {
+		private static readonly Operation ReadOperation = new Operation(Plugins.Authorization.Operations.Node.Gossip.Read);
```

Comment:
The `ReadOperation` here is referring to the same resource as the cluster gossip (the resource being `node/gossip`) which means that a client strictly has access to the cluster gossip read.
Would it make more sense to introduce a new resource `client/gossip` or something similar?

cc: @jageall